poses:
  pose_inicial:
    plat_angle: 0.0
    plat_duration: 0.0
    manip_angle: 1.0
    manip_duration: 60.0
  pose_baixo:
    plat_angle: 0.0
    plat_duration: 0.0
    manip_angle: -1.0
    manip_duration: 60.0
  pose_pegar_bloco:
    plat_angle: 0.0
    plat_duration: 0.0
    manip_angle: -1.0
    manip_duration: 59.0
  pose_deposito_1:
    plat_angle: 160.0
    plat_duration: 2.0
    manip_angle: 50.0
    manip_duration: 2.0
  pose_deposito_2:
    plat_angle: 180.0
    plat_duration: 2.0
    manip_angle: 50.0
    manip_duration: 2.0
  pose_deposito_3:
    plat_angle: 200.0
    plat_duration: 2.0
    manip_angle: 50.0
    manip_duration: 2.0
